pkg/controller/config-controller.go

```diff
@@ -156,6 +162,47 @@ func (r *CDIConfigReconciler) reconcileStorageClass(config *cdiv1.CDIConfig) err
 	return nil
 }
 
+func (r *CDIConfigReconciler) reconcileDefaultPodResourceRequirements(config *cdiv1.CDIConfig) error {
+	log := r.Log.WithName("CDIconfig").WithName("DefaultPodResourceRequirements")
+
+	// TODO: set as default value (0)
```

Comment:
I think this TODO is done, I also think the log is not needed as its normal operation.

Reply:
Done!